### PR TITLE
Result API endpoints

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Fixes: #966

I do not want to spend much time on the values that require a connection between multiple models because I am finishing the refactoring of the scheme in another PR (#954). I will update/enhance those values once the refactor is merged.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
@IceWreck Can you please take if a brief look if the values/format will be usable from the dashboard side?

**Comment by lachmanfrantisek** [comment]
rechcek

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by IceWreck** [comment]
Yeah, one more thing, in the `/copr-builds` and `/koji-builds` endpoints but not the `/srpm-builds` endpoints can you add `packit_build_id` (not sure about this name) for all chroots ?

I'm talking about the `/copr-builds` endpoint here, not `/copr-builds{id}`

example entry 
```
{
    "project": "packit-packit-1108-stg",
    "build_id": "1959353",
    "status_per_chroot": {
      "fedora-rawhide-x86_64": "success",
      "fedora-32-x86_64": "success",
      "fedora-33-x86_64": "success"
    },
    "packit_build_id": {
      "fedora-rawhide-x86_64": 8830,
      "fedora-32-x86_64": 8831,
      "fedora-33-x86_64": 8832
    }
    "build_submitted_time": "12/02/2021 11:20:01",
    "web_url": "https://copr.fedorainfracloud.org/coprs/build/1959353/",
    "ref": "6c6f10c488056c5f2e0d1bc05e5e09c4d4535bce",
    "pr_id": 1108,
    "branch_name": null,
    "repo_namespace": "packit",
    "repo_name": "packit",
    "project_url": "https://github.com/packit/packit"
  },
```
And a similar thing for `/koji-builds` as well.

**Comment by lachmanfrantisek** [comment]
@IceWreck OK, good point. I'll try to enhance that.

**Comment by lachmanfrantisek** [comment]
@IceWreck Do you want to do this for koji builds as well? They are currently not grouped at all.

**Comment by IceWreck** [comment]
I'm sorry I forgot they weren't grouped. Just add something like `"packit_build_id": 3100`.

**Comment by lachmanfrantisek** [comment]
Done, I've used `packit_id` and `packit_id_per_chroot` to not need to use a different name for tests and builds.

File: packit_service/models.py [hunk 1]
```diff
@@ -650,6 +650,12 @@ def get_branch_name(self) -> Optional[str]:
             return trigger_object.name
         return None
 
+    def get_release_tag(self) -> Optional[int]:
+        trigger_object = self.job_trigger.get_trigger_object()
+        if isinstance(trigger_object, ProjectReleaseModel):
+            return trigger_object.tag_name
```
Comment (TomasTomecek) [inline packit_service/models.py:656]: tag_name is an int? interesting and confusing to me

**Review by TomasTomecek — APPROVED** [review]
pretty good: will be awesome to have these in the dashboard!

some new models methods confused me a bit, otherwise LGTM

File: packit_service/models.py (hunk 1 above)
Comment (lachmanfrantisek) [reply packit_service/models.py:656]: Ups. I've copy-pasted an already existing method..;) Will fix that..;)

**Comment by lachmanfrantisek** [comment]
> some new models methods confused me a bit, otherwise LGTM

I've done some refactoring of those `build/test -> pr/release/..` methods in my next PR.

File: packit_service/models.py (hunk 1 above)
Comment (lachmanfrantisek) [reply packit_service/models.py:656]: And mypy does not complain?

Comment (lachmanfrantisek) [reply packit_service/models.py:656]: Fixed.

**Review by jpopelka — COMMENTED** [review]
LGTM (I know I'm late for the party)

**Pull request merged**